utils/models.py

```diff
@@ -0,0 +1,12 @@
+from typing import Any
+
+from django.db import models
+
+
+class BaseModel(models.Model):
+    class Meta:
+        abstract = True
+
+    def save(self, *args: Any, **kwargs: Any) -> None:  # pylint: disable=signature-differs
```

Comment:
If signature is different than it is originally please fix it. As far as I remember in `save` method signature was also `full_clean` parameter.

Reply:
Done. I based my previous solution on [this](https://github.com/PyCQA/pylint-django/issues/94) for the same issue. I found some voices that this shouldn't be raised by pylint. Also if I'm not wrong in the future version of pylint this won't be raised if `*args` and `**kwargs` are passed.

Reply:
I'm not sure if pylint won't raise problem about it because it's quite important to always have same unchanged signature because in python we don't recognize different method based on signature as it is done for example in c++ or java